hack/gen-swagger-doc/deploy.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/bash
+
+set -e
+
+GITHUB_FQDN=github.com
+API_REF_REPO=kubevirt-incubator/api-reference
+API_REF_DIR=/tmp/api-reference
+
+git clone \
+    "https://${API_REFERENCE_PUSH_TOKEN}@${GITHUB_FQDN}/${API_REF_REPO}.git" \
+    "${API_REF_DIR}"
+cp -f hack/gen-swagger-doc/html5/content/*.html "${API_REF_DIR}/content/"
```

Comment:
I think we need to delete the content of `${API_REF_DIR}/content/` first, to not collect left over files in the git repo over time.

Reply:
Done